charts/pipeline/templates/deployment-worker.yaml

```diff
@@ -81,6 +81,15 @@ spec:
             secretKeyRef:
               name: {{ include "pipeline.fullname" . }}
               key: githubToken
+        # Pipeline service and TLS
```

Comment:
I don't think the builtin worker needs these settings.

Reply:
Currently not, but it works as a reference for the future worker. We can remove this for now. (The tls secret was mounted here also before this PR).

Reply:
Removed.